doc/panes/historylog.rst

```diff
@@ -41,6 +42,8 @@ You might need to show invisible files in order to see it on a non-Windows opera
 .. image:: /images/history/history-log-file.png
    :alt: Spyder History Log file
 
+While there is currently no built-in way to clear history from the Spyder interface aside from resetting preferences, you can do so by closing Spyder, deleting this file and restarting it again.
```

Comment:
```suggestion
While there is currently no built-in way to clear history from the Spyder interface aside from resetting preferences, you can do so by closing Spyder, deleting this file and restarting Spyder again.
```
I know it sounds repetitive but the last noun before this was "this file" so it is a bit confusing. What do you think?

Reply:
Great catch and excellent point, thanks.